guides/common/modules/ref_best-practices-for-sync-plans.adoc

```diff
@@ -0,0 +1,22 @@
+[id="best-practices-for-sync-plans_{context}"]
+= Best practices for sync plans
+
+* Add sync plans to products and regularly synchronize content to avoid overly large changes in repositories.
+{Team} recommends synchronizing content rather each night than only once a month.
+* {Team} recommends distributing synchronization tasks over several hours to reduce the task load by creating multiple sync plans with *Custom Cron*.
+* Use *Custom Cron* to specify recurring synchronization using cron expressions.
```

Comment:
I'd raise this with the content team. IMHO Katello should be enhanced to figure this out by itself. Placing the burden on the user isn't the best UX.